dexios/src/domain/decrypt.rs

```diff
@@ -57,10 +57,18 @@ where
         Some(header_reader) => {
             let (header, aad) = Header::deserialize(&mut *header_reader.borrow_mut())
                 .map_err(|_| Error::DeserializeHeader)?;
-            req.reader
-                .borrow_mut()
-                .seek(std::io::SeekFrom::Start(header.get_size()))
-                .map_err(|_| Error::DeserializeHeader)?;
+
+            // Try reading an empty header from the content.
```

Comment:
I'm not too sure this should be the behaviour - I don't think the file should be forced to have an empty header if the user wants to decrypt from a detached one.

What's the reasoning behind this? Hopefully I can try to understand this better :)

Reply:
![image](https://user-images.githubusercontent.com/7839004/179764468-fa7c9e24-d3e2-422c-9528-61699eaebd5d.png)

Well for the user we save an extra 416 bytes)

Why do we need to put the header into a separate file if dexios still allocates memory for the header in the encrypted file?

Reply:
> Why do we need to put the header into a separate file if dexios still allocates memory for the header in the encrypted file?

To allow for easy re-attaching later on, but this way makes sense too :)